src/bin/rustfmt.rs

```diff
@@ -9,17 +9,37 @@
 // except according to those terms.
 
 #![cfg(not(test))]
+#![feature(path_ext)]
 
 extern crate rustfmt;
 
 use rustfmt::{WriteMode, run};
 
-use std::fs::File;
-use std::io::Read;
+use std::env;
+use std::fs::{File, PathExt};
+use std::io::{self, Read, Error, ErrorKind};
+use std::path::PathBuf;
+
+fn lookup_config_file() -> io::Result<PathBuf> {
```

Comment:
I guess it could change name in the future, I had planned to use default.toml as the config file *we* provide, and then the user would provide rustfmt.toml (or something) in their project directory which would override the defaults in default.toml.